packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -57,7 +61,7 @@ exports.sourceNodes = async (
   })
   const allData = await Promise.all(
     _.map(data.data.links, async (url, type) => {
-      if (type === `self` || type === `describedby`) return
+      if (_.includes(disallowedLinkTypes, type)) return
```

Comment:
for my benefit: why do we use lodash here instead of the native [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) method?
this never runs in a browser right?

Reply:
Thank you, @larowlan :)

Reply:
@larowlan lot of this code was originally written when Gatsby supported node version that didn't have `Array.prototype.includes` (node 4), but now we support Node 8+ (And soon when Node 8 reaches end of support life we will be Node 10+). You are free to use `includes` method in newly written code

Reply:
For what it's worth, I reused lodash here since the file did not know what node supported. I can update to use the native method. That way new code slowly move away from lodash

Reply:
This has been updated to use `Array.prototype.includes`